doc/content/lorawan/regional-limitations-of-rf-use/_index.md

```diff
@@ -0,0 +1,65 @@
+
+# US (902-928 MHz)
+
+The following regulations are applied to the USA, Canada, and all other countries in ITU Region 2 adopting the entire [FCC](https://www.fcc.gov/) (Federal Communications Commission) 47 CFR Part 15 regulations in the 902-928 ISM band.
+
+In the United States, the ISM band is divided into **80 channels**: 72 for uplink and 8 for downlink.
+
+- There are **64 uplink channels**, numbered from 0 to 63, each with a bandwidth of 125 kHz. These channels start at a center frequency at 902.3 MHz and increment linearly by 200 kHz up to 914.9 MHz.
+- There are **8 uplink channels**, numbered from 64 to 71, each with a bandwidth of 500 kHz. These channels start at a center frequency at 903.0 MHz and increment linearly by 1.6 MHz up to 914.2 MHz.
+- There are **8 downlink channels**, numbered from 0 to 7, each with a bandwidth of 500 kHz. These channels start at a center frequency at 923.3 MHz and increment linearly by 600 kHz up to 927.5 MHz.
+
+The end devices must have the capability to store parameters for the 72 uplink channels mentioned above.
+
+The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that the devices are able to successfully transmit data while avoiding interference with other devices.
+
+The value of dwell time for LoRaWAN for the US902-928 band is 400ms. This means that the device should spend 400 ms on each channel before switching to the next channel.
+
+When designing and operating devices in the US ISM band, it is important to take this dwell time limitation into account. For example, if a device is transmitting data at a high rate, it may need to use a faster hopping rate in order to comply with the 400 ms dwell time limitation.
+
+Both end devices and gateways are also limited to a maximum EIRP of +30 dBm. The maximum EIRP can be maintained by adjusting the transmitter power, antenna gain, or cable loss. 
+
+The downlink channel for the RX1 receive window is based on the uplink channel that was used to send the preceding uplink. The default downlink channel for the RX2 receive window is 923.3 MHz with a data rate of DR8.
+
+For Class B beacons, the US902-928 band uses 8 frequencies 923.3, 923.9, 924.5, 925.1, 925.7, 926.3, 926.9, 927.5 starting at 923.3 MHz on a rotation basis. For LoRaWAN relays, channel 0 uses 916.7 MHz and 919.9 MHz for WOR transmissions respectively, and channel 1 uses 918.3 MHz and 921.5 MHz for WOR ACK transmissions respectively.
+
+It is also important to note that there may be additional regulatory requirements and certifications that must be met in order to legally operate in the ISM band in specific countries. It is highly recommended to have your LoRaWAN end device [certified](https://lora-alliance.org/lorawan-certification/) by the LoRa Alliance® before releasing it to the market.
```

Comment:
LoRa Alliance certification is not good enough. You device also needs to be FCC/ICASA/RED/R&TTE/etc approved/certified.

For a quick summary see: https://www.element.com/nucleus/2018/wireless-devices-a-guide-to-compliance-in-the-us-and-eu
and https://electronics.stackexchange.com/a/186516/225812